pkg/apis/extensions/types.go

```diff
@@ -773,3 +773,277 @@ type ReplicaSetCondition struct {
 	// +optional
 	Message string
 }
+
+// +genclient
```

Comment:
drop the changes to this file

Reply:
this got added in this commit, then removed in the next. please squash or fixup to avoid noise